context.go

```diff
@@ -336,38 +336,62 @@ func (c *Context) Rem(d, x, y *Decimal) (Condition, error) {
 }
 
 // Sqrt sets d to the square root of x.
+// Sqrt uses the Babylonian method for computing the square root, which uses
```

Comment:
I'm okay with that. I didn't realize the change in position would have godoc effects.